cdk-v2/Vagrantfile

```diff
@@ -36,8 +36,16 @@ SUBSCRIPTION_INFO = "
     $ export SUB_PASSWORD=password
   "
 
-NO_SERVICE_MANAGER_ERROR = "vagrant-service-manager plugin is not installed, run `vagrant plugin install vagrant-service-manager` to install the plugin."
-NO_VAGRANT_REGISTRATION_ERROR = "vagrant-registration plugin is not installed, run `vagrant plugin install vagrant-registration` to install the plugin."
+REQUIRED_PLUGINS = %w(vagrant-service-manager vagrant-registration)
+message = ->(name) { "#{name} plugin is not installed, run `vagrant plugin install #{name}` to install it." }
+errors = []
+# Validate and collect error message if plugin is not installed
+REQUIRED_PLUGINS.each {|plugin| errors << message.call(plugin) unless Vagrant.has_plugin?(plugin) }
+if errors.size > 1
+  raise errors.inject(''){|msg, e| msg += "\n * #{e}" }
```

Comment:
Any reason why you raise a `RuntimeError`? The old code used a VagrantError which gives much nicer output. In your case, I get:

    There was an error loading a Vagrantfile. The file being loaded
    and the error message are shown below. This is usually caused by
    a syntax error.

    Path: /Users/hardy/work/developer-platform/openshift-cdk/cdk-v2/Vagrantfile
    Line number: 47
    Message: RuntimeError: vagrant-registration plugin is not installed, run `vagrant plugin install vagrant-registration` to install it.

Not only is the actual reason just a single line at the end, but the information at the beginning is even misleading "There was an error loading a Vagrantfile. The file being loaded and the error message are shown below. This is usually caused by a syntax error.". There is no error in the Vagrantfile in this case.

Using VagrantError you just get:

    vagrant-registration plugin is not installed, run `vagrant plugin install vagrant-registration` to install the plugin.